app/controllers/concerns/hyku_addons/doi_form_behavior.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+module HykuAddons
+  module DOIFormBehavior
+    extend ActiveSupport::Concern
+
+    included do
+      self.terms += [:doi]
+
+      delegate :doi, to: :model
+    end
+
+    def primary_terms
+      !Flipflop.doi_tab? && Flipflop.doi_minting? ? super << :doi : super
```

Comment:
Instead of a ternary, could this be simplified a little by using a guard statement?

```ruby
def primary_terms
    return super if Flipflop.doi_tab? && !Flipflop.doi_minting?

    super << :doi
end
```

I'd usually argue that ternary conditions are good for `this or that`, rather than adding logic with side effects.

Reply:
Absolutely